2022-03-20 13:14:51.608 INFO  [acme.Launcher@main] Starting Launcher using Java 1.8.0_281 on DESKTOP-UVONV50 with PID 16272 (C:\Users\juanq\Desktop\Workspace-22.0\Frameworks\Acme-Framework-22.1\target\classes started by juanq in C:\Users\juanq\Desktop\Workspace-22.0\Projects\Acme-Toolkit) 
2022-03-20 13:14:51.609 DEBUG [acme.Launcher@main] Running with Spring Boot v2.4.2, Spring v5.3.3 
2022-03-20 13:14:51.610 INFO  [acme.Launcher@main] The following profiles are active: development,populator 
2022-03-20 13:14:54.060 DEBUG [org.apache.jasper.servlet.JasperInitializer@main] Initializing Jasper for context [application] 
2022-03-20 13:14:55.610 ERROR [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean@main] Failed to initialize JPA EntityManagerFactory: @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit 
2022-03-20 13:14:55.611 WARN  [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@main] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit 
2022-03-20 13:14:55.652 ERROR [org.springframework.boot.SpringApplication@main] Application run failed at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit\nat org.hibernate.cfg.ToOneFkSecondPass.doSecondPass(ToOneFkSecondPass.java:97): @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit
2022-03-20 13:14:55.653 ERROR [acme.Launcher@main] org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1788): Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit\norg.hibernate.cfg.ToOneFkSecondPass.doSecondPass(ToOneFkSecondPass.java:97): @OneToOne or @ManyToOne on acme.artifact.PartOf.toolkit references an unknown entity: java.awt.Toolkit 
2022-03-20 13:14:55.655 INFO  [acme.Launcher@main] The application log provides additional debugging messages! 
2022-03-20 13:14:56.668 DEBUG [acme.Launcher@main] Exit code = 1 
